### PR TITLE
Add option to update the rcu schema password after the population by …

### PR DESCRIPTION
**Pull request by jshum2479**
Added a flag called -updateRCUSchemaPassword.  When this flag is specified in RCUDBInfo in the domainInfo section, the password will take precedence over the one in the shadow component_schema_info table created by rcu the first time.

### CODE REVIEW

File: core/src/main/python/wlsdeploy/util/cla_utils.py [hunk 1]
```diff
@@ -70,6 +70,7 @@ class CommandLineArgUtil(object):
     ATTRIBUTES_ONLY_SWITCH     = '-attributes_only'
     FOLDERS_ONLY_SWITCH        = '-folders_only'
     RECURSIVE_SWITCH           = '-recursive'
+    UPDATE_RCU_SCHEMA_PASS_SWITCH = '-updateRCUSchemaPassword'
```
Comment (ddsharpe) [inline core/src/main/python/wlsdeploy/util/cla_utils.py]: Why is this new switch necessary?  Why isn't the presence of the new domainInfo attribute sufficient?

File: core/src/main/python/update.py [hunk 2]
```diff
@@ -44,6 +46,20 @@
 from wlsdeploy.util.model import Model
 from wlsdeploy.util.weblogic_helper import WebLogicHelper
 from wlsdeploy.util import model as model_helper
+# from wlsdeploy.aliases.model_constants import RCU_DB_INFO
```
Comment (ddsharpe) [inline core/src/main/python/update.py]: Why is it necessary to leave all these commented sections of code?  Why not remove them?

File: core/src/main/python/update.py [hunk 3]
```diff
@@ -184,6 +201,81 @@ def __process_encryption_args(optional_arg_map):
         optional_arg_map[CommandLineArgUtil.PASSPHRASE_SWITCH] = String(passphrase)
     return
 
+# def __process_update_rcu_schema_password_args(model, model_context, aliases):
```
Comment (ddsharpe) [inline core/src/main/python/update.py]: Why is it necessary to leave all these commented sections of code? Why not remove them?

File: core/src/main/python/wlsdeploy/util/cla_utils.py (hunk 1 above)
Comment (jshum2479) [reply core/src/main/python/wlsdeploy/util/cla_utils.py]: This is in WDT which has no idea whether the information is new or old.

File: core/src/main/python/update.py (hunk 3 above)
Comment (jshum2479) [reply core/src/main/python/update.py]: I will remove the commented out code

**Pull request merged**